### PR TITLE
testing the platform io setup with vscode

### PR DESCRIPTION
**Pull request by Puncture-source**
here is a description that would describe it more

### CODE REVIEW

File: test2/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  // put your setup code here, to run once:
+  int result = myFunction(2, 3);
+}
+
+void loop() {
+  // put your main code here, to run repeatedly:
+}
+
+// put function definitions here:
+int myFunction(int x, int y) {
```
Comment (Hunter-dJ03) [inline test2/src/main.cpp:16]: CALL THIS ADD NOT MYFUNCTION

**Review by Hunter-dJ03 — CHANGES_REQUESTED** [review]
DONT CALL YOUR MAIN FILE MAIN

FIX THESE CHANGE

File: test2/src/main.cpp (hunk 1 above)
Comment (Puncture-source) [reply test2/src/main.cpp:16]: all done

**Review by Hunter-dJ03 — APPROVED** [review]
GOOD WORK

**Pull request merged**